integration-tests/tests/mocked/get-question/MockConfigFile.json

```diff
@@ -0,0 +1,196 @@
+{
+  "StateMachines": {
+    "get-question": {
+      "TestCases": {
+        "HappyPathTest": {
+          "Get NINO": "GetUserNINOSuccess",
+          "GET Questions from HMRC": "GetHMRCQuestionsSuccess",
+          "Get unanswered questions": "GetUnansweredQuestionsSuccess"
+        },
+        "ZeroUnansweredQuestions": {
+          "Get NINO": "GetUserNINOSuccess",
+          "GET Questions from HMRC": "GetHMRCQuestionsSuccess",
+          "Get unanswered questions": "GetZeroUnansweredQuestionsSuccess"
+        },
+        "InValidRequestSessionIdNotPresent": "",
+        "InValidRequestNinoIdNotPresent": {
+          "Get NINO": "GetUserNINOUnSuccessfull"
+        }
+      }
+    }
+  },
+  "MockedResponses": {
+    "GetUserNINOSuccess": {
+      "0": {
+        "Return": {
+          "Count": 1,
+          "Items": [
+            {
+              "sessionId": {
+                "S": "12345"
+              },
+              "nino": {
+                "S": "AA000003D"
+              }
+            }
+          ],
+          "ScannedCount": 1
+        }
+      }
+    },
+    "GetHMRCQuestionsSuccess": {
+      "0": {
+        "Return": {
+          "sessionId": "12345",
+          "nino": "AA000003D"
+        }
+      }
+    },
+    "GetUnansweredQuestionsSuccess": {
+      "0": {
+        "Return": {
+          "Count": 6,
```

Comment:
Can we limit the mock to 1 question? This will reduce the size and complexity of this test as we are only checking that questions exist and returning the first one